src/wallet/db.cpp

```diff
@@ -633,3 +639,44 @@ bool CDB::PeriodicFlush(std::string strFile)
 
     return ret;
 }
+
+bool CWalletDBWrapper::Rewrite(const char* pszSkip)
+{
+    return CDB::Rewrite(*this, pszSkip);
+}
+
+bool CWalletDBWrapper::Backup(const std::string& strDest)
+{
+    if (!env)
```

Comment:
Inconsistent style with using/skipping braces (above, you used braces, here, you don't). I'd add them.